DataFormats/Detectors/ITSMFT/common/include/DataFormatsITSMFT/ClusterPattern.h

```diff
@@ -0,0 +1,80 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file ClusterTopology.h
+/// \brief Definition of the ClusterTopology class.
+///
+/// \author Luca Barioglio, University and INFN of Torino
+///
+/// Short ClusterPattern descritpion
+///
+/// This class contains the information of the pattern of a cluster of pixels (bitmask), together with the number of
+/// rows and columns of the vinding box.
+///
+
+#ifndef ALICEO2_ITSMFT_CLUSTERPATTERN_H
+#define ALICEO2_ITSMFT_CLUSTERPATTERN_H
+#include <Rtypes.h>
+#include <array>
+#include <iosfwd>
+#include "DataFormatsITSMFT/Cluster.h"
+
+namespace o2
+{
+namespace ITSMFT
+{
+class ClusterTopology;
+class TopologyDictionary;
+class BuildTopologyDictionary;
+
+class ClusterPattern
+{
+ public:
+  /// Default constructor
+  ClusterPattern();
+  /// Standard constructor
+  ClusterPattern(int nRow, int nCol, const unsigned char patt[Cluster::kMaxPatternBytes]);
+  /// Returns the pattern
+  std::array<unsigned char, Cluster::kMaxPatternBytes + 2> getPattern() const { return mBitmap; }
```

Comment:
You add `+2` to a constant in multiple places. I think it would be more readable to define another constant which has the meaning of `kMaxPatterBytes + 2`.

Reply:
Thank you: done